synedrion/src/cggmp21/params.rs

```diff
@@ -123,6 +127,21 @@ pub trait SchemeParams: Debug + Clone + Send + PartialEq + Eq + Send + Sync + 's
     /// plus one bit (so that any curve scalar still represents a positive value
     /// when treated as a 2-complement signed integer).
     type Paillier: PaillierParams;
+
+    /// Returns ``true`` if the parameters satisfy a set of inequalities
+    /// required for them to be used for the CGGMP scheme.
+    fn are_self_consistent() -> bool {
```

Comment:
I think `is_` is more correct, as in "is this (one) set of params self-consistent"? Alternatively just drop the verb and call it `self_consistent() -> bool`.